adapters/pubmatic/pubmatic_test.go

```diff
@@ -158,23 +159,39 @@ func TestExtractPubmaticExtFromRequest(t *testing.T) {
 		request *openrtb2.BidRequest
```

Comment:
As a unit test, I think that this test could use one more test case that exercises the line that sets `pmReqExt.Marketplace`.

Reply:
added test case for **Valid Pubmatic marketplace ext**